parser/helper/helpers.go

```diff
@@ -180,12 +180,12 @@ func (h *Helper) FilterTxsByActorType(ctx context.Context, txs []*types.Transact
 		addrTo, err := address.NewFromString(tx.TxTo)
 		if err != nil {
 			h.logger.Sugar().Errorf("could not parse address. Err: %s", err)
-			return nil, err
```

Comment:
I don't really like this, as if something fails, we loose the tx at all, and never know. I guess this is the general approach on the fil parser, so I am ok now. But in the future we need a way to track these errors, as we won't never be able to fix them otherwise.

Reply:
agree, we need to include some metrics here